src/reducers/score-reducer.ts

```diff
@@ -0,0 +1,12 @@
+import { ACTION, InitialReducer } from "../components/Game/SinglePlayer/PlayArea/PlayArea";
+
+export function scoreReducer(state:InitialReducer,action:ACTION){
+    switch(action.type){
+        case "CORRECT_ANSWER":
+            return {...state,totalScore:state.totalScore+action.payload};
```

Comment:
code formatting can be done.
```suggestion
            return {...state, totalScore: state.totalScore + action.payload};
```